receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper_others.go

```diff
@@ -18,11 +18,11 @@ func (s *scraper) recordCPUTimeMetric(now pcommon.Timestamp, cpuTime *cpu.TimesS
 func (s *scraper) recordCPUUtilization(now pcommon.Timestamp, cpuUtilization ucal.CPUUtilization) {}
 
 func getProcessName(processHandle, string) (string, error) {
-	return nil, nil
+	return "", nil
 }
 
 func getProcessExecutable(processHandle) (string, error) {
-	return nil, nil
+	return "", nil
```

Comment:
Sorry for nit picking, but this is unrelated. Please submit another PR is needed

Reply:
Done
- https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/24379